{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2021-02-08",
    "AffectedVersionRange": "*",
    "Summary": "Arbitrary Code Execution",
    "Contributor": {
        "Discloser": "61599526",
        "Fixer": ""
    },
    "Package": {
        "Registry": "pip",
        "Name": "labml",
        "URL": "https://pypi.org/project/labml",
        "Downloads": "136"
    },
    "CWEs": [
        {
            "ID": "94",
            "Description": "Arbitrary Code Execution"
        }
    ],
    "CVSS": {
        "Version": "3.1",
        "AV": "",
        "AC": "",
        "PR": "",
        "UI": "",
        "S": "",
        "C": "",
        "I": "",
        "A": "",
        "E": "",
        "RL": "",
        "RC": "",
        "Score": "8.5"
    },
    "CVEs": [
        ""
    ],
    "Repository": {
        "URL": "https://github.com/lab-ml/labml",
        "Codebase": [
            "Jupyter Notebook"
        ],
        "Owner": "lab-ml",
        "Name": "labml",
        "Forks": "39",
        "Stars": "417",
        "ForkName": "labml"
    },
    "Permalinks": [
        "https://drive.google.com/file/d/1bOoFb46vvOyMcGLLYi_0ixgKFJiaYG1m/view?usp=sharing"
    ],
    "References": [
        {
            "Description": "",
            "URL": ""
        }
    ],
    "PrNumber": "1858",
    "FixSubmissionCount": 1
}